rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,109 @@
+
+
+### User Stories
+
+* As a member of a chartered organization of the DCP, I would like to reduce the amount of time I spend writing up an RFC and move more quickly to execution while still ensuring that I receive actionable feedback and advice for improvement.
+
+## Detailed Design
+
+### 1. Add additional RFC states
+
+The current possible states of an RFC are:
+ 
+ 1. "rfc-community-review"
+ 2. "rfc-oversight-review" 
+ 3. "rfc-approved"
+ 4. "rfc-declined" 
+
+In practice, there have been cases where an RFC is no longer implemented (i.e. "deprecated") or an RFC has been withdrawn due to either incompleteness or having been published too early. 
+
+Additionally, with the encouraged behavior of submitting draft pull requests prior to RFC readiness for review, there is no way to clearly designate the fact that the RFC should not be reviewed at all until the PR is no longer a draft.
+
+We propose adding the following new labels: 
+
+**rfc-draft** : This indicates that the authors are still actively working on the RFC, that the RFC is in a draft state, and readers should **not** comment on the RFC yet. Additionally this state serves to indicate more clearly the backlog of RFCs in development to avoid duplication of effort of overlapping RFCs and to help elicit offers of co-authorship.
+ 
+Currently "pre-RFCs" are often worked on in private Google Docs. To indicate more clearly the to the community the backlog of RFCs in development, avoid duplication of effort of overlapping RFCs and to help encourage co-authorship and early collaboration, draft RFCs should post a title and problem statement or motivation. They need not be full fledged RFCs with every section populated.
+
+**rfc-deprecated** : This indicates that the design is no longer implemented or executed in the DCP and has been usurped by a newer design or process. Note that the deprecated label is not standalone; it may be used in conjunction with  "rfc-approval" label for example.
+
+**rfc-paused** : This indicates that an RFC has been paused for further refinement. This roughly implies that an RFC has gone from being in a review status to a "pre-rfc" status and should not be reviewed further until designated so.
+
+**rfc-withdrawn** : This label is applied when an RFC has not finished the review process (and therefore has not been  explicitly rejected) but has been voluntarily withdrawn.
+
+Like the existing RFC process, any change of state to the RFC must be announced on the DCP slack channel. Also note that labeling an RFC has no implications on whether the PR is open or closed. Generally, RFCs in a terminal state (i.e. "approved", "declined", or "withdrawn").
+
+### 2. Reduce the RFC minimum review time
+
+The current RFC process requires a minimum of 2 weeks in DCP Community Review followed by 1 week of DCP Oversight Review. The intention of the "Community" review is to receive feedback not from other DCP team members, but rather the  scientific community.
```

Comment:
_The intention ... but rather the scientific community_ is an incorrect statement. Currently, community review includes team members and users. For example, community review is the time for developers to review a governance RFC.  I think that you could strike the _intention_ sentence. You have good points below that many RFCs are published when there's already consensus among the proposers which would allow us to shorten review period to two weeks.